### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by alma-castant**
Contact page. Responding to this issue: https://github.com/systers/portal/issues/78

I added fontawesome to the project for the social media icons

### CODE REVIEW

File: systers_portal/static/css/style.css [hunk 1]
```diff
@@ -134,6 +134,23 @@ body > .container {
   }
 }
 
+
+/* Common: about us
+-------------------------------------------------- */
+.contact-text {
+ font-family: Georgia, "Times New Roman", Times, serif;
```
Comment (ana-balica) [inline systers_portal/static/css/style.css]: I see there's a single indentation space. Could you make it 2 spaces?

File: systers_portal/static/css/style.css [hunk 2]
```diff
@@ -134,6 +134,23 @@ body > .container {
...
+ font-size: 1.2em;
+ line-height: 1.8em;
+ margin: 20px 0;
+}
+
+.contact-icon{
```
Comment (ana-balica) [inline systers_portal/static/css/style.css]: Please add a space before `{`.

File: systers_portal/static/css/style.css [hunk 3]
```diff
@@ -134,6 +134,23 @@ body > .container {
...
+  margin: 0 5px;
+}
+
+.contact-social a{
```
Comment (ana-balica) [inline systers_portal/static/css/style.css]: And a space here before `{`.
Also add a hover effect to the links, maybe a darker color.

File: systers_portal/templates/common/contact.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  <div class="container col-md-8 col-md-offset-2">
+    <h1>Contact Systers</h1>
+    <hr>
+    <p class="contact-text">
+      Systers is curated by the current <a href="mailto:systers-keeper@systers.org">Systers-keeper</a>, Rose Robinson.
+    </p>
+    <p class="contact-text">
+      You can also connect with Systers on our social networks.
+    </p>
+    <div class="container-fluid contact-social">
```
Comment (ana-balica) [inline systers_portal/templates/common/contact.html]: Hmm, this bootstrap class `container-fluid` adds some padding to left and right. I think it doesn't look so good with this offset from the left. I assume the class can be safely removed or substituted with another CSS class.

File: systers_portal/templates/common/contact.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      <a href="https://www.facebook.com/abi.systers"><i class="fa fa-facebook-official fa-2x contact-icon"></i></a>
+      <a href="https://twitter.com/systers_org"><i class="fa fa-twitter fa-2x contact-icon"></i></a>
+      <a href="https://www.linkedin.com/groups?home=&gid=3593"><i class="fa fa-linkedin fa-2x contact-icon"></i></a>
```
Comment (ana-balica) [inline systers_portal/templates/common/contact.html]: The `&` symbol generated an HTML validation error. To remove the error, use `&amp;` instead.

File: systers_portal/templates/common/contact.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    <h1>Contact Systers</h1>
+    <hr>
+    <p class="contact-text">
+      Systers is curated by the current <a href="mailto:systers-keeper@systers.org">Systers-keeper</a>, Rose Robinson.
+    </p>
+    <p class="contact-text">
+      You can also connect with Systers on our social networks.
+    </p>
+    <div class="container-fluid contact-social">
+      <a href="https://www.facebook.com/abi.systers"><i class="fa fa-facebook-official fa-2x contact-icon"></i></a>
```
Comment (ana-balica) [inline systers_portal/templates/common/contact.html]: Please make the URLs to open in a new tab with `target="_blank"`. In that way users won't leave Systers Portal right away.

**Comment by alma-castant** [comment]
@ana-balica I updated the request

**Comment by ana-balica** [comment]
@alma-castant Everything looks very good.

We have some major changes on our develop branch (because of the renaming from community_admin to admin) and when merging this pull request there are conflicts. Would it be possible for you to rebase this branch locally against the latests changes from develop and solve the conflicts? It's rather trivial. If you can't do it, tell me and I will do it myself.

**Comment by alma-castant** [comment]
@ana-balica It should be solved now. I amend the previous commit though, so I had to commit again

**Pull request merged**